packages/integration-karma/test/services/hooks/index.spec.js

```diff
@@ -0,0 +1,72 @@
+import { createElement } from 'lwc';
```

Comment:
```suggestion
import { createElement, register } from 'lwc';
```

Reply:
The compiler prevents from doing this. This is why I extract it from the IIFE.